mapie/classification.py

```diff
@@ -161,7 +162,7 @@ def _check_parameters(self) -> None:
         if self.method not in self.valid_methods_:
             raise ValueError(
                 "Invalid method. "
-                "Allowed values are 'score'."
+                "Allowed values are 'score' or 'cumulated_score'."
```

Comment:
Supprimer la mention "(to be implemented)" dans la docstring décrivant les méthodes.

Reply:
supprimé